frontend/src/components/CsetsDataTable.js

```diff
@@ -101,6 +103,71 @@ function CsetsDataTable(props) {
     </div>
   );
 }
+
+/* Copied from CsetsDataTable(). This table only shows the selected csets. */
+function CsetsSelectedDataTable(props) {
```

Comment:
This review is more for Siggie than I, but I just skimmed through this implementation/use of `CsetsSelectedDataTable` for my own familiarity.